src/routes/blog/index.svelte

```diff
@@ -12,10 +25,22 @@
   import Spacing from '$lib/components/sections/spacing.svelte';
 
   export let posts: Newsletter[] = [];
+
+  async function filterPosts(event: CustomEvent) {
+    const tags = event.detail;
+
+    window.history.replaceState({}, '', `${window.location.pathname}?l=${tags.join(',')}`);
+
+    const target = new URL('/blog.json', window.location.origin);
+    target.searchParams.set('l', tags.join(','));
+
+    const response = await fetch(target.toString());
+    posts = await response.json();
+  }
 </script>
 
 <svelte:head>
-  <title>acmCSUF / README</title>
+  <title>Blog | ACM at CSUF</title>
```

Comment:
I think Mike decided to use `/` for the title separator.

Reply:
Thanks 4 letting me know. I will change it to `/`.